lib/Widgets/DynamicNotebook.vala

```diff
@@ -668,6 +695,45 @@ namespace Granite.Widgets {
         // Use temporary field to avoid breaking API this can be dropped while preparing for 0.4
         string _add_button_tooltip;
 
+        public string add_button_tooltip_markup {
+            owned get { return add_button.get_tooltip_markup (); }
+            set { add_button.tooltip_markup = value; }
+        }
+
+        /**
+         * The text shown in the close button tooltip
+         */
+        private string _close_button_tooltip = _("Close Tab");
+        public string close_button_tooltip {
+            get { return _close_button_tooltip; }
+            set {
+                if (value != _close_button_tooltip) {
+                    tabs.foreach ((t) => {
+                        t.close_button_tooltip = value;
+                    });
+                }
+
+                _close_button_tooltip = value;
+            }
+        }
+
+        /**
```

Comment:
I think these comments need to be right above the public properties to show in the docs